lib/msal-common/src/error/ClientAuthError.ts

```diff
@@ -42,6 +42,10 @@ export const ClientAuthErrorMessage = {
         code: "blank_guid_generated",
         desc: "The guid generated was blank. Please review the trace to determine the root cause."
     },
+    invalidStateError: {
+        code: "invalid_state",
+        desc: "State was not valid, please check the network trace."
```

Comment:
Is "network trace" the right place to be looking?

Reply:
Changed.